newspaper/extractors.py

```diff
@@ -38,16 +38,21 @@
 COLON_SPLITTER = StringSplitter(":")
 SPACE_SPLITTER = StringSplitter(' ')
 NO_STRINGS = set()
-A_REL_TAG_SELECTOR = "a[rel=tag]"
+
+A_REL_TAG_SELECTOR = "a[rel=tag], li[class~='css-1og0r8'] a span, a p[class~='css-m9s1mv'], div[class~='c-article-regular__tags-item']"
 A_HREF_TAG_SELECTOR = ("a[href*='/tag/'], a[href*='/tags/'], "
                        "a[href*='/topic/'], a[href*='?keyword=']")
 RE_LANG = r'^[A-Za-z]{2}$'
 
+xpaths_to_remove = ['//*[@id="main"]/div[2]/div[2]/div[1]/div[2]',
+                    '//*[@id="main"]/div[2]/div[2]/div[1]/figure/div[2]/figcaption' #e24
+                    ]
+
 good_paths = ['story', 'article', 'feature', 'featured', 'slides',
               'slideshow', 'gallery', 'news', 'video', 'media',
               'v', 'radio', 'press']
 bad_chunks = ['careers', 'contact', 'about', 'faq', 'terms', 'privacy',
-              'advert', 'preferences', 'feedback', 'info', 'browse', 'howto',
+              'advert', 'preferences', 'feedback', 'info', 'browse', 'howto'
```

Comment:
@bajo1207 I'm working on merging this PR into a fork. Was the removal of this comma, thereby making the string 'howtoaccount' intentional? If so can you explain?